lib/addons/karpenter/index.ts

```diff
@@ -13,7 +13,26 @@ import * as sqs from 'aws-cdk-lib/aws-sqs';
 import { Rule } from 'aws-cdk-lib/aws-events';
 import { SqsQueue } from 'aws-cdk-lib/aws-events-targets';
 import { Cluster } from 'aws-cdk-lib/aws-eks';
+import { EbsDeviceVolumeType } from 'aws-cdk-lib/aws-ec2';
 
+interface BlockDeviceMapping {
+    deviceName?: string;
+    virtualName?: string;
+    ebs?: Ebs;
+    noDevice?: string;
+}
+  
+interface Ebs {
```

Comment:
Ebs is a bit too bold for this interface  as it appears to represent ElasticBlockStorage as a service. How about either "EbsVolumeSettings` | `EbsVolumeMappings` or just `EbsMappings`?

Reply:
thanks for the feedback. I've renamed to EBSVolumeMapping